plugins/woocommerce-blocks/tests/e2e/tests/templates/single-product-template.block_theme_with_templates.spec.ts

```diff
@@ -2,13 +2,12 @@
  * External dependencies
  */
 import { test, expect } from '@woocommerce/e2e-playwright-utils';
-import { BLOCK_THEME_WITH_TEMPLATES_SLUG } from '@woocommerce/e2e-utils';
 
 const testData = {
 	permalink: '/product/belt',
 	templateName: 'Single Product Belt',
 	templatePath: 'single-product-belt',
-	templateType: 'wp_template',
+	templateType: 'wp_template' as 'wp_template' | 'wp_template_part',
```

Comment:
Considering that we are using the `wp_template | wp_template_part` in multiple places, I wonder if wouldn't be better to define a type or an Enum for it, for example:

```typescript
enum TemplateType {
  WpTemplate = "wp_template",
  WpTemplatePart = "wp_template_part",
}
```

Reply:
Good point! I created a `TemplateType` type in d67619be6c4787f4fb04731b8797b28d40cbf751. I don't have a strong preference between an enum and a type, though. I went with a type as there are only two values and it seemed to add less complexity, but I wouldn't mind changing it.